dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/AssuranceData.kt

```diff
@@ -9,8 +8,7 @@ import org.dataland.datalandbackend.model.enums.eutaxonomy.AssuranceOptions
  * Level of assurance for the reported data and information about the assurance provider
  */
 data class AssuranceData(
-    @field:JsonProperty(required = true)
```

Comment:
That's an interesting one. Let's discuss it. 
Should we
a) make eveything in AssuranceData Optional or 
b) make AssuranceData non-optional, but if it's present, it should contain at least a bit of information.